### ISSUE TITLE
Be able to define beginning of week

### ISSUE THREAD

**Issue by thec0keman** [supremebeing7/time_log_robot#32]
It might be convenient to be able to have more control over the default time span.  At a minimum it would be nice to define what the first day of the week is.  However, it may also be nice to say something like 'the default can be x number of days ago'.

**Comment by Parmandil** [comment]
Yeah, my ideal default is that it always goes back 7 days.

**Comment by supremebeing7** [comment]
Makes sense. If anyone wants to tackle this, might dovetail nicely with https://github.com/supremebeing7/time_log_robot/issues/41
